datalad_debian/configure_builder.py

```diff
@@ -47,10 +72,17 @@ class ConfigureBuilder(Interface):
             args=('spec',),
             metavar='property=value',
             nargs='*',
-            doc="""Stuff to complete the template"""),
+            doc="""Values to complete placeholders in the specified
+             template"""),
     )
 
-    _examples_ = []
+    _examples_ = [
+        dict(text="Configure the default Singularity recipe in the builder "
+                  "subdataset, from a distribution superdataset",
```

Comment:
"from a distribution superdataset". Do you mean that this command is "executed from a distribution superdataset", or that the configuration is somehow from a distribution superdataset?